docs/getting-started/preview.md

```diff
@@ -0,0 +1,79 @@
+# W4H-Preview
+## Introduction
+This document is for the preview of W4H toolkit. W4H Toolkit is an ecosystem that covers most of the common features needed when using wearables in a clinical study or or trial. As studies can have very different requirements, W4H toolkit is designed to be flexible and incrementally adoptable. This toolkit provides the following components:
+
+
+**Dashboard** : An intuitive interface that offers data input forms and visual analytics tools.
+
+**Notebooks**: A collection of sample Jupyter notebooks that showcase various data analysis techniques. These include Pandas notebook & Pyspark notebooks.
+
+**Database**: A curated postgres database specifically used in W4H examples, providing real-world data for testing and demonstration purposes.
+
+## Docker Image
+The W4H Toolkit is available as a Docker image, which can be easily pulled from [Docker Hub](https://hub.docker.com/r/uscimsc/w4h). This containerized version includes all the necessary components and dependencies, making it simple to set up and use the toolkit.
+
+### Pulling the W4H Docker Image
+To get started with the W4H Toolkit, follow these steps:
+1. Ensure Docker is installed on your system.
+2. Open a terminal or command prompt.
+3. Pull the W4H Docker image from Docker hub by running the following command:
+```bash
+docker pull usc-imsc:w4h-toolkit-preview
+```
+
+### Running the container
+Once the image is pulled, run the container using following command:
+```bash
+docker run -e POSTGRES_DB=postgres -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=postgres -p 8501:8501 -p 5432:5432 -p 8888:8888 --user root -v $(pwd):/app usc-imsc:w4h-toolkit-preview
+```
+
+## Accessing the Dashboard
```

Comment:
all this is very good but we just want a top-level list with each item basic information, links, what it does like for the dashboard link to home allows to do x, link to import allows to do y, ... again this should be in the demo and paper.